ui/tests/acceptance/client-count-history-test.js

```diff
@@ -19,7 +19,7 @@ const tokenWithPolicy = async function(name, policy) {
   return consoleComponent.lastLogOutput;
 };
 
-module('Acceptance | usage metrics', function(hooks) {
+module('Acceptance | client count history', function(hooks) {
```

Comment:
I think this is already on your todos but we should make sure to add another test for client count current -- also clicking the tabs to go between the two